core/Cargo.toml

```diff
@@ -6,102 +6,102 @@ edition = "2021"
+sc-block-builder = { workspace = true, default-features = true }
+sc-transaction-pool-api = { workspace = true, default-features = true }
+sc-transaction-pool = { workspace = true, default-features = true }
+sc-consensus-aura = { workspace = true, default-features = true }
+node-primitives = { workspace = true, default-features = true }
 
 # Substrate frame dependencies
-frame-system = { workspace = true }
-frame-support = { workspace = true }
-pallet-babe = { workspace = true }
-frame-benchmarking = { workspace = true }
+frame-system = { workspace = true, default-features = true }
+frame-support = { workspace = true, default-features = true }
+pallet-babe = { workspace = true, default-features = true }
+frame-benchmarking = { workspace = true, default-features = true }
 
 # Polkadot dependencies
-polkadot-core-primitives = { workspace = true }
-polkadot-node-primitives = { workspace = true }
-polkadot-primitives = { workspace = true }
-polkadot-parachain-primitives = { workspace = true }
-polkadot-runtime-parachains = { workspace = true }
-polkadot-service = { workspace = true }
-polkadot-overseer = { workspace = true }
+polkadot-core-primitives = { workspace = true, default-features = true }
+polkadot-node-primitives = { workspace = true, default-features = true }
+polkadot-primitives = { workspace = true, default-features = true }
+polkadot-parachain-primitives = { workspace = true, default-features = true }
+polkadot-runtime-parachains = { workspace = true, default-features = true }
+polkadot-service = { workspace = true, default-features = true }
+polkadot-overseer = { workspace = true, default-features = true }
 ## Currently only needed to make cumulus-relay-chain-inprocess-interface compile, not sure why cumulus works though
-polkadot-cli = { workspace = true }
+polkadot-cli = { workspace = true, default-features = true }
 
 # Cumulus dependencies
-cumulus-primitives-parachain-inherent = { workspace = true }
-cumulus-relay-chain-interface = { workspace = true }
-cumulus-primitives-core = { workspace = true }
-cumulus-client-parachain-inherent = { workspace = true }
+cumulus-primitives-parachain-inherent = { workspace = true, default-features = true }
+cumulus-relay-chain-interface = { workspace = true, default-features = true }
+cumulus-primitives-core = { workspace = true, default-features = true }
+cumulus-client-parachain-inherent = { workspace = true, default-features = true }
 
 [dev-dependencies]
-polkadot-test-runtime = { workspace = true }
+polkadot-test-runtime = { workspace = true, features = ["std"] }
 
-staging-parachain-info = { workspace = true }
+staging-parachain-info = { workspace = true, features = ["std"] }
 
-sp-consensus-beefy = { workspace = true }
-frame-benchmarking = { workspace = true }
-pallet-timestamp = { workspace = true }
-pallet-balances = { workspace = true }
+sp-consensus-beefy = { workspace = true, features = ["std"] }
+frame-benchmarking = { workspace = true, features = ["std"] }
```

Comment:
Do we need this dependency?

Reply:
We use it here:
https://github.com/centrifuge/fudge/blob/polkadot-v1.7.2/core/src/provider/mod.rs#L128

Not sure if we can drop it from here, maybe @mustermeiszer can answer that?

Reply:
We never compile fudge with `frame-benchmarking` so the benchmarking version at least is never used in `centrifuge-chain`

Reply:
Not sure. Remove if not needed.

Reply:
Last commit has the removal of this, we can revert that if needed. Thank you!